doc/auth.md

```diff
@@ -0,0 +1,71 @@
+# Enable Authentication
+
+Client can communicate with Pravega in a more secure way by enabling Authentication.
+Authentication Handler can be implemented in different ways. For details see [Pravega AUthentication](https://github.com/pravega/pravega/blob/master/documentation/src/docs/auth/auth-plugin.md) for details.
+
+PasswordAuthHandler(default) is the most basic authentication handler.
+For using this, user needs to do the following:
+
+1. Create a userPassword file containing `<user>:<password>:<acl>;` on each line for one user.
+Delimiter should be `:` with `;` at the end of each line.
+Use the   [PasswordCreatorTool](https://github.com/pravega/pravega/blob/master/controller/src/test/java/io/pravega/controller/auth/PasswordFileCreatorTool.java) to create a new file with the password encrypted. Use this file when creating the secret in next step.
+
+Example:
+
+cat userPassword.txt
+```
+admin:353030303a633132666135376233353937356534613430383430373939343839333733616463363433616532363238653930346230333035393666643961316264616661393a3639376330623663396634343864643262663335326463653062613965336439613864306264323839633037626166663563613166333733653631383732353134643961303435613237653130353633633031653364366565316434626534656565636335663666306465663064376165313765646263656638373764396361:*,READ_UPDATE;
```

Comment:
A scrollbar shows up due to the length of this text. Please reformat into multiple lines for better readability.

Reply:
IMO it should be fine to see the scroll bar.
Formatting into multiple lines would be very misleading, since this file is supposed to have one user per line.